INSTALL.md

```diff
@@ -0,0 +1,67 @@
+# How to install MacGitver
+
+In present state, you need to compile MacGitver from source. We provide a compilable version from our default branch `development`. If you like to help with creating binary packages, please create a [pull request](https://github.com/macgitver/MacGitver/pulls) or contact me via [mail](mailto:nils@macgitver.org).
+
+## Prerequisites
+
+To be able to build and install, the following packages are required:
+
+* Qt >=5.4
+* CMake >=3.1
+* C++11 (libc++)
+* A recent C++ compiler GCC (>=4.9) or Clang (>=3.6)
```

Comment:
I haven't played with GCC 5.1 much yet. But it might be more up to date nowadays... I see no reason to ditch it though.